metacat-connector-hive/build.gradle

```diff
@@ -13,11 +13,14 @@
 
 dependencies {
     compile project(':metacat-common-server')
-    compile "org.apache.hive:hive-metastore:${hive_version}"
+    compile("org.apache.hive:hive-metastore:${hive_version}") {
+        exclude group: 'com.google.guava'
+    }
+    compile("com.facebook.presto.hive:hive-apache:${facebook_hive_apache}")
```

Comment:
The HiveStorageFormat.java needs the class in the jar. We can put in the hive-exec jar, which is a fat jar causing dependencies issue.